application/btsieve/config/bitcoin_missing.toml

```diff
@@ -0,0 +1,8 @@
+[ethereum]
```

Comment:
🙃 I'd suggest to use the same terminology than in the api tests "ethereum_only" to be more consistent.